### DISCUSSION TITLE
Setup recommendation - from scratch

### DISCUSSION THREAD

**Discussion by vascorey** [blakeblackshear/frigate#6749] [Ask A Question]
Hello friends!

I am new to Frigate and I have to say I am very impressed with the features - it feels like a very good improvement on Synology DS Cam, which I was using before (kudos to everyone involved / @blakeblackshear).

I would like your advice on a setup I want to do for my home - I will have 9 indoor or outdoor cameras, ideally PTZ (Frigate still can't handle PTZ, right?), most of the cameras will be connected to an ethernet PoE switch, though the outdoor cameras may be connected with Wifi.

I think 1080p for the cams should be sufficient. 

I have some machines lying around, not in use, which I can use for frigate (possibly in exclusivity):
* a lenovo i7; 16GB RAM; 512GB SSD laptop w/ Ubuntu installed
* a Raspberry Pi 4
* I can rent a server, e.g., from hetznner
These machines can be on or off premises - I know how to setup port forwarding to connect to the cameras from the Internet. I can have these machines 100% dedicated to Frigate NVR, though having Home Assistant would be a plus.

I would like to connect these machines also to an alarm system - the idea is to check whether the alarm is triggered to notify me if a person is detected by the NVR. 

Taking this into account, I would like to ask for for advice on:
1) Which hardware?
 - cams, ideally PoE powered / outdoors with Wifi
 - PCs - if the ones are not appropriate
 - Google Coral AI (USB? How many?)
2) Which setup (.conf file)? E.g., how many CPUs should I dedicate to handling the cams (I think it's one per, but I think I have 8 cores...)
3) Which software would you recommend to do the alarm system integration?
4) - For bonus points - Any recommendations on alarms compatible with home assistant that I can setup together with this?

Hope to hear from you soon,
V

**Comment by NickM-27** [comment]
>  (Frigate still can't handle PTZ, right?)

PTZ is supported in 0.13 (unreleased, currently in dev branch only)

> a lenovo i7; 16GB RAM; 512GB SSD laptop w/ Ubuntu installed

you didn't specify which generation i7 but assuming it is 6th gen or newer then you could use OpenVINO which would obviate the need for a coral

> Taking this into account, I would like to ask for for advice on:
> 
> 1. Which hardware?
> 
> * cams, ideally PoE powered / outdoors with Wifi

The frigate docs have some general camera recommendations, I'd follow those brands for all cases

> * PCs - if the ones are not appropriate
> * Google Coral AI (USB? How many?)

If you have an intel iGPU that is compatible with OpenVINO then there is likely no need for a coral. That being said, the amount of corals depends on the amount of activity you plan to have at once. 

> 2. Which setup (.conf file)? E.g., how many CPUs should I dedicate to handling the cams (I think it's one per, but I think I have 8 cores...)

I don't see a reason to mess with this unless some other application you are running calls for it.

> 3. Which software would you recommend to do the alarm system integration?

There may be better ones but frigate has great integration with home assistant and that is the only one I would be familiar with so there could be better ones for your usecase.

**Comment by vascorey** [comment]
Thank you for your answers!
Cheers
